product_docs/docs/efm/4/efm_user/09_controlling_logging.mdx

```diff
@@ -8,7 +8,7 @@ legacyRedirectsGenerated:
 
 <div id="controlling_logging" class="registered_link"></div>
 
-Failover Manager writes and stores one log file per agent and one startup log per agent in `/var/log/<cluster_name>-4.2` (where `<cluster_name>` specifies the name of the cluster).
+Failover Manager writes and stores one log file per agent and one startup log per agent in `/var/log/<cluster_name>-4.<x>`.
```

Comment:
This is actually wrong and it should be just /var/log/efm-4.<x>

The cluster name will affect the name of log files, but the directory is always the above.